app/src/components/widgets/singleLevelWidgets/charts/passingRatePerLaunch/passingRatePerLaunch.jsx

```diff
@@ -14,20 +14,56 @@
  * limitations under the License.
  */
 
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 import { messages } from 'components/widgets/common/messages';
+import { FAILED, PASSED, INTERRUPTED, SKIPPED, IN_PROGRESS } from 'common/constants/testStatuses';
+import { ALL } from 'common/constants/reservedFilterIds';
+import { getDefaultTestItemLinkParams } from 'components/widgets/common/utils';
+import { statisticsLinkSelector } from 'controllers/testItem';
+import { activeProjectSelector } from 'controllers/user';
+import { STATS_PASSED } from 'common/constants/statistics';
 import { PassingRateChart } from '../common/passingRateChart';
 
 const getFilterName = ({ contentParameters, content: { result = {} } = {} } = {}) =>
   `${contentParameters.widgetOptions.launchNameFilter} #${result.number}`;
 
-export const PassingRatePerLaunch = (props) => (
-  <PassingRateChart
-    {...props}
-    filterNameTitle={messages.launchName}
-    filterName={getFilterName(props.widget)}
-  />
-);
-PassingRatePerLaunch.propTypes = {
-  widget: PropTypes.object.isRequired,
-};
+@connect(
+  (state) => ({
+    project: activeProjectSelector(state),
+    getStatisticsLink: statisticsLinkSelector(state),
+  }),
+  {
+    navigate: (linkAction) => linkAction,
+  },
+)
+export class PassingRatePerLaunch extends Component {
+  static propTypes = {
+    getStatisticsLink: PropTypes.func.isRequired,
+    navigate: PropTypes.func.isRequired,
+    project: PropTypes.string.isRequired,
+    widget: PropTypes.object.isRequired,
+  };
+
+  onChartClick = (data) => {
+    const { widget, getStatisticsLink, project } = this.props;
+    const launchId = widget.content.result.id;
+    const link = getStatisticsLink({
+      statuses: data.id === STATS_PASSED ? [PASSED] : [FAILED, INTERRUPTED, SKIPPED, IN_PROGRESS],
```

Comment:
It is redundant to send status `IN_PROGRESS` here, because the items with this status are not affects statistics and widgets.